### PR TITLE
fix: Add Amit Kumar to the list

### PR DESCRIPTION
**Pull request by UltimateBrains**

### CODE REVIEW

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]
@UltimateBrains the issue number doesn't seem to be correct and seems to be someone else's. Could you please open a new issue with you image url in the description and then link the issue number to the PR? Please tell us if you need help. Thanks (:

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
@UltimateBrains I don't think you have opened an issue.
- Open an issue, and call it `add UltimateBrains`.
- Paster your image in: use this `profile` .
- Open the issue.
- Get the issue number, and change this to the correct issue num.

If you need any more help, feel free to ask.

**Comment by UltimateBrains** [comment]
Okay,Let me check

**Review by CodingSpecies — COMMENTED** [review]
Thank you for adding the issue, and the number.

File: data.json [hunk 1]
```diff
@@ -1547,4 +1547,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/56837137/139258468-f7b162b5-94f2-4fb3-94e0-7da43db795dd.png",
     "issueNumber": 805
   }
+   {
```
Comment (CodingSpecies) [inline data.json:1550]: Please can you format line `1550`, so the brace is in line? @UltimateBrains 
After that, it should be good.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Review by CodingSpecies — APPROVED** [review]
Thank you @UltimateBrains LGTM.

File: data.json [hunk 2]
```diff
@@ -1546,5 +1546,11 @@
     "githubUsername": "mayankchaudhary26",
     "imageUrl": "https://user-images.githubusercontent.com/56837137/139258468-f7b162b5-94f2-4fb3-94e0-7da43db795dd.png",
     "issueNumber": 805
+  },
+  {
+    "name": "Amitkumar",
```
Comment (Panquesito7) [inline data.json:1551]: Shouldn't your name be `Amit Kumar`?
```suggestion
    "name": "Amit Kumar",
```

**Review by shreyaansjain06 — CHANGES_REQUESTED** [review]
Please do changes suggested by @Panquesito7

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Please check @Panquesito7's comments

**Review by krishguptadev — COMMENTED** [review]
## Merge Conflict

Thank you for your contribution. Your pull request currently has conflicting changes with our `main` branch. In order to review this pull request, we will need those conflicts to be resolved.

We recommend doing this with the `git rebase` command. If you are unsure of how to fix this, let us know and we would be happy to assist you.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by krishguptadev — COMMENTED** [review]
LGTM!

**Pull request merged**